### PR TITLE
add upgrade step from 0.11.x to 1.0.0

### PR DESCRIPTION
**Pull request by youngbupark**
Thank you for helping make the Dapr documentation better!

If you are a new contributor, please see the [this contribution guidance](https://docs.dapr.io/contributing/contributing-docs/) which helps keep the Dapr documentation readable, consistent and useful for Dapr users.

Note that you must see that the suggested changes do not break the website [docs.dapr.io](https://docs.dapr.io). See [this overview](https://github.com/dapr/docs/blob/master/README.md#overview) on how to setup a local version of the website and make sure the website is built correctly.

In addition, please fill out the following to help reviewers understand this pull request:

## Description

add upgrade step from 0.11.x to 1.0.0

## Issue reference

_Please reference the issue this PR will close: #908

### CODE REVIEW

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md [hunk 1]
```diff
@@ -193,6 +193,60 @@ APP ID     APP PORT  AGE  CREATED
...
+#### Upgrade from 0.11.x to 1.0.0
+
+Upgrading from 0.11.x to 1.0.0 safely requires additional step to keep 0.11.x placement service to migrate from 0.11.x placement service to 1.0.0 placement service. These two commands disables helm upgrade to uninstall 0.11.x placement service.
+
+```sh
+kubectl annotate deployment dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+kubectl annotate svc dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+```
+
+Then [ex]port certs manually](exporting-certs-manually).
```
Comment (yaron2) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md:205]: misformat?

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md (hunk 1 above)
Comment (youngbupark) [reply daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md]: fixed

**Comment by youngbupark** [comment]
@orizohar @msfussell Please review this and feel free to edit it.

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md [hunk 2]
```diff
@@ -193,6 +193,61 @@ APP ID     APP PORT  AGE  CREATED
 nodeapp    3000      16h  2020-07-29 17:16.22
 ```
 
+#### Upgrade from 0.11.x to 1.0.0
```
Comment (orizohar) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md:196]: Are these instructions only from 0.11 to 1.0.0 or do they also apply from 0.10 / 0.9? (if so we should change this)
Also, just to make sure users don't think this refers to k8s version, could be good to say "Upgrade from Dapr 0.11.x to 1.0.0"

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md [hunk 3]
```diff
@@ -193,6 +193,61 @@ APP ID     APP PORT  AGE  CREATED
...
+
+```sh
+dapr mtls export -o ./certs
+```
+
+Upgrade Dapr using the below commands:
+
+```sh
+helm repo update
+helm upgrade dapr dapr/dapr --version 1.0.0-rc1 --namespace dapr-system --reset-values --set-file dapr_sentry.tls.root.certPEM=./certs/ca.crt --set-file dapr_sentry.tls.issuer.certPEM=./certs/issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=./certs/issuer.key --set global.ha.enabled=true --wait
```
Comment (orizohar) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md:215]: This explicitly says `--version 1.0.0-rc1` will we need to update these instructions on the next RCs? perhaps it's better to change the section header to "Upgrade ... to 1.0.0-rc1"

Comment (msfussell) [reply daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md:215]: Or you can say, pick the v1.0 release version e.g. --version 1.0.0-rc1

**Review by msfussell — COMMENTED** [review]
LGTM

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md (hunk 3 above)
Comment (tcnghia) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md:215]:
```suggestion
helm upgrade dapr dapr/dapr --version 1.0.0-rc.1 --namespace dapr-system --reset-values --set-file dapr_sentry.tls.root.certPEM=./certs/ca.crt --set-file dapr_sentry.tls.issuer.certPEM=./certs/issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=./certs/issuer.key --set global.ha.enabled=true --wait
```

**Comment by tcnghia** [comment]
Instruction works for me

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md (hunk 2 above)
Comment (youngbupark) [reply daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md:196]: I have tested 0.11 only. technically the upgrade from 0.10, 0.9 to 1.0.0 should work.

Comment (youngbupark) [reply daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md]: > could be good to say "Upgrade from Dapr 0.11.x to 1.0.0"
make sense. I updated it.

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md (hunk 3 above)
Comment (youngbupark) [reply daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md]: Please review it again :)

**Review by yaron2 — DISMISSED** [review]
lgtm

**Review by orizohar — APPROVED** [review]

**Pull request merged**